packages/uicore/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/uicore",
-  "version": "3.99.3",
+  "version": "3.99.4",
```

Comment:
should the minor version be changed as a new prop is added?

Reply:
It's not breaking. So should be fine as per the MFE pattern versioning that we're doing